dist/Math-Complex/Makefile.PL

```diff
@@ -0,0 +1,22 @@
+{ use 5.006; }
+
+use ExtUtils::MakeMaker;
+
+WriteMakefile(
+	      NAME            => 'Math::Complex',
+	      VERSION_FROM    => 'lib/Math/Complex.pm',
+	      INSTALLDIRS     => 'perl',
```

Comment:
INSTALLDIRS should be conditional on perl version:
```
INSTALLDIRS => ( $] < 5.011 ? 'perl' : 'site' ),
```

Reply:
Sorry for the duplication, but I thought a comment on each PR would make it easier to track.

Reply:
Fixed.